src/Compilers/CSharp/Test/Semantic/Semantics/InterceptorsTests.cs

```diff
@@ -21,20 +23,21 @@ namespace Microsoft.CodeAnalysis.CSharp.UnitTests.Semantics;
 
 public class InterceptorsTests : CSharpTestBase
 {
-    private static readonly (string, string) s_attributesSource = ("""
+    private static readonly (string text, string path) s_attributesSource = ("""
         namespace System.Runtime.CompilerServices;
 
         [AttributeUsage(AttributeTargets.Method, AllowMultiple = true, Inherited = false)]
         public sealed class InterceptsLocationAttribute : Attribute
         {
-            public InterceptsLocationAttribute(string filePath, int line, int character)
-            {
-            }
+            public InterceptsLocationAttribute(string filePath, int line, int character) { }
+            public InterceptsLocationAttribute(int version, string data) { }
```

Comment:
Just checking, we're tracking the API update on the runtime side too?

Reply:
Runtime has not shipped this attribute. In practice source generators use file-local declarations of it instead. We probably need to submit it to runtime for review though so that it can ship in the framework as part of the stable feature release.